azure-devops/ecommerce-projects/99_main.tf

```diff
@@ -0,0 +1,58 @@
+terraform {
+  required_version = ">= 1.1.5"
+  required_providers {
+    azuredevops = {
+      source  = "microsoft/azuredevops"
+      version = ">= 0.2.0"
+    }
+    azurerm = {
+      version = ">= 2.98.0"
+    }
+  }
+  backend "azurerm" {}
+}
+
+provider "azurerm" {
+  features {}
+}
+
+provider "azurerm" {
```

Comment:
what is the purpose of following providers ?

Reply:
the goal is to provide access to azure subscriptions (DEV/UAT/PROD) where the keyvault dedicated to the domain is present, in this case pagopa-ecommerce-kv